Functions.Templates/Bindings/bindings-bundle-v4.json

```
@@ -1859,6 +1859,118 @@
           "help": "$signalROut_connectionString_help"
         }
       ]
+    },
+    {
+      "type": "sql",
+      "displayName": "$sqlIn_displayName",
+      "direction": "in",
+      "enabledInTryMode": false,
+      "documentation": "$content=Documentation\\sqlIn.md",
+      "extension": {
+        "id": "Microsoft.Azure.WebJobs.Extensions.Sql", "version": "1.0.122-preview"
+      },
+      "settings": [
+        {
+          "name": "name",
+          "value": "string",
+          "defaultValue": "object",
+          "required": true,
+          "label": "$sqlIn_name_label",
+          "help": "$sqlIn_name_help",
+          "validators": [
+            {
+              "expression": "^[a-zA-Z][a-zA-Z0-9]{0,127}$",
+              "errorText": "[variables('parameterName')]"
+            }
+          ]
+        },
+        {
+          "name": "commandText",
+          "value": "string",
+          "defaultValue": "Select * from object",

```

I've never really found a consistent default I like, but having at least stuff that matches our samples seems like a good idea

```suggestion
          "defaultValue": "SELECT * FROM Products",

```